### PR TITLE
7.2 Initial implementation of classes.

### PR DESCRIPTION
**Pull request by TonyLiu836**
Partly finished #7 , initial implementation of some basic functionality. Pls roast 🙏

### CODE REVIEW

**Comment by RunpengNie** [comment]
Please include the issue number in the PR's description.

File: GetFit/src/main/java/com/TheTroisMousquetaires/GetFit/entity/User.java [hunk 1]
```diff
@@ -1,4 +1,67 @@
 package com.TheTroisMousquetaires.GetFit.entity;
 
 public class User {
+    String name;
```
Comment (RunpengNie) [inline GetFit/src/main/java/com/TheTroisMousquetaires/GetFit/entity/User.java]: Make `name` public, and all the other attributes should be private.

**Comment by RunpengNie** [comment]
Overall, it is a good design and implementation.

**Review by RunpengNie — CHANGES_REQUESTED** [review]

**Comment by RunpengNie** [comment]
The other minor change: change the branch name to `7-Design-Program-Logic-2`. Make sure the only special character we have in branch names is `-`.

**Review by RunpengNie — APPROVED** [review]
LGTM! Now we need one more review from Joe @Junzhou-712  or Vincent @VincentXiao997 .

**Review by Junzhou-712 — APPROVED** [review]
LGTM! Thanks

**Pull request merged**